lib/plugins/aws/deploy/lib/createStack.js

```diff
@@ -31,7 +31,16 @@ module.exports = {
 
   createStack() {
     const stackName = `${this.serverless.service.service}-${this.options.stage}`;
-
+    if (/^[^a-zA-Z].+|.*[^a-zA-Z0-9\-].*/.test(stackName) || stackName.length > 128) {
+      const errorMessage = [
+        `The stack name "${stackName}" is not quallify. `,
```

Comment:
Could you change it to `The stack service name "${stackName}" is not valid`? Because in the `serverless.yml` we call it service.